src/PhpDoc/PhpDocNodeResolver.php

```diff
@@ -317,15 +316,28 @@ public function resolveReturnTag(PhpDocNode $phpDocNode, NameScope $nameScope):
 
 	public function resolveThrowsTags(PhpDocNode $phpDocNode, NameScope $nameScope): ?\PHPStan\PhpDoc\Tag\ThrowsTag
 	{
-		$types = array_map(function (ThrowsTagValueNode $throwsTagValue) use ($nameScope): Type {
-			return $this->typeNodeResolver->resolve($throwsTagValue->type, $nameScope);
-		}, $phpDocNode->getThrowsTagValues());
+		$resolved = null;
+
+		foreach (['@throws', '@phpstan-throws'] as $tagName) {
```

Comment:
This should be reversed - in case of both `@throws` and `@phpstan-throws` being present, `@phpstan-throws` should win. Also please add a test for this situation.